### DISCUSSION TITLE
Telegram Bots That Use The Joke API

### DISCUSSION THREAD

**Discussion by friedelinehustenkraehe** [Sv443/JokeAPI#296] [Q&A] [answered]
Does anyone know Telegram bots that use the Joke API?

**Comment by Sv443** [comment]
Not that I know of but if you do find or create any, I would love if you add them as a dependent by [submitting an issue](https://github.com/Sv443/JokeAPI/issues/new/choose), so other people can discover them

**Comment by CrazyMarvin** [comment, marked as answer]
@TECH-SAVVY-GUY and myself are working on a Telegram bot that uses this Joke API.  

Telegram bot: https://t.me/FunnyTelegramBot
Source: https://github.com/Crazy-Marvin/FunnyTelegramBot

**Reply by tech-savvy-guy** [reply]
In response to @CrazyMarvin's reply, I was thinking if @Sv443 could provide us with an Authorization token, so as to bypass the limit on the number of requests to the API?

**Reply by Sv443** [reply]
@TECH-SAVVY-GUY @CrazyMarvin if you get in touch with me (preferably over [Discord](https://dc.sv443.net)) I'll give you a token.

**Reply by tech-savvy-guy** [reply]
![image](https://user-images.githubusercontent.com/83786816/137591724-5e428196-cb14-41d9-ab9b-516562712215.png)

Request sent :)
